scripts/test/shared.py

```diff
@@ -229,7 +229,7 @@ def has_shell_timeout():
     '--wasm-staging',
     '--experimental-wasm-eh',
     '--experimental-wasm-simd',
-    '--experimental-wasm-anyref',
+    '--experimental-wasm-externref',
```

Comment:
Should we use this opportunity to also change this to --experimental-wasm-reftypes` to match browser flags since I think the rest of the proposal is implemented too under the same flag?

Reply:
sounds good to me